lib/helpers/mail_sender.js

```diff
@@ -0,0 +1,29 @@
+import { mailAuthPassword, mailAuthUser } from '../config/mail_sender';
+
+import nodemailer from 'nodemailer';
+import { rollbar } from '../config/rollbar';
+
+const transport = nodemailer.createTransport({
+  service: 'gmail',
+  auth: {
+    user: mailAuthUser,
+    pass: mailAuthPassword,
+  },
+});
+
+const buildMessage = (userEmail, subject, html) => {
+  return {
+    from: mailAuthUser,
+    to: userEmail,
+    subject: subject,
+    html: html,
```

Comment:
Un detalle pavo, cuando la clave se llama igual que el parámetro/variable/const, se puede omitir:


```suggestion
    subject,
    html,
```